crates/sui/src/sui_commands.rs

```diff
@@ -149,10 +149,20 @@ impl SuiCommand {
 
                 swarm.launch().await?;
 
-                let mut interval = tokio::time::interval(std::time::Duration::from_secs(5));
+                let mut interval = tokio::time::interval(std::time::Duration::from_secs(3));
+                let mut unhealthy_cnt = 0;
                 loop {
                     for node in swarm.validators_mut() {
-                        node.health_check(true).await?;
+                        if let Err(err) = node.health_check(true).await {
+                            unhealthy_cnt += 1;
+                            if unhealthy_cnt > 3 {
```

Comment:
we are not doing what the comment says, the internal ticks in the `loop` but the unhealthy_cnt can be incremented to 3 in one `for` loop

Reply:
That is a very good catch!